### PR TITLE
Minor event module and other changes

### PR DESCRIPTION
**Pull request by Yash-10**
- Reference for the eclipse was added.
- Docstring for node event was missing earlier, here it is added.

I also wonder if `EclipseEvent` should be made an abstract class since objects of `EclipseEvent` don't have any useful existence, right? In fact, its `__call__` method just returns the position vector of the secondary body, which doesn't seem to be related to any event.

### CODE REVIEW

File: src/poliastro/ephem.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ class Ephem:
     """Time history of position and velocity of some object at particular epochs.
 
     Instead of creating Ephem objects directly,
-    you are encouraged to use the available classmethods.
+    the user is encouraged to use the available classmethods.
```
Comment (astrojuanlu) [inline src/poliastro/ephem.py:155]:
```suggestion
    use the available classmethods.
```

?

File: src/poliastro/twobody/events.py [hunk 2]
```diff
@@ -221,9 +221,23 @@ def __call__(self, t, u_, k):
...
+        i.e. event is triggered during both crossings.
+
+    """
+
     def __init__(self, terminal=False, direction=0):
         super().__init__(terminal, direction)
 
     def __call__(self, t, u_, k):
         self._last_t = t
+        # Check if z-coordinate of the satellite is zero or not.
```
Comment (astrojuanlu) [inline src/poliastro/twobody/events.py:242]:
```suggestion
        # Check if the z coordinate of the satellite is zero
```

?

**Review by astrojuanlu — COMMENTED** [review]
Good improvements! Added a couple of comments

**Comment by Yash-10** [comment]
Thanks for those suggestions!

**Review by astrojuanlu — APPROVED** [review]
🚀

**Pull request merged**